### PR TITLE
contributors.csv: added new ORCID column

### PR DESCRIPTION
**Pull request by neteler**
This PR adds a new `orcid` column (https://orcid.org/) to the file for better citation of authors.
For discussion, see https://lists.osgeo.org/pipermail/grass-psc/2022-January/002531.html

I have made a quick round of searches to identify some ORCIDs. Please review.

### CODE REVIEW

File: contributors.csv [hunk 1]
```diff
@@ -1,66 +1,66 @@
...
+-,Ivan Shmakov,<ivan theory.asu.ru>,Russia,1gray,yes,
+-,Eric Patton,<eric.patton canada.ca>,Canada,epatton,yes,
+-,Laura Toma,<ltoma bowdoin.edu>,USA,ltoma,yes,
+-,Markus Metz,<markus.metz.giswork googlemail.com>,Germany,mmetz,yes,0000-0002-4038-8754
+-,Maris Nartiss,<maris.gis gmail.com>,Latvia,marisn,yes,
+-,Marco Pasetti,<marco.pasetti alice.it>,Italy,marcopx,yes,
+-,Yann Chemin,<yann.chemin gmail.com>,Philippines,ychemin,yes,0000-0001-9232-5512
+-,Colin Nielsen,<colin.nielsen gmail.com>,USA,cnielsen,yes,
+-,Anne Ghisla,<a.ghisla gmail.com>,Italy,aghisla,yes,
+-,Helmut Kudrnovsky,<hellik web.de>,Austria,hellik,yes,0000-0001-6622-7169
```
Comment (hellik) [inline contributors.csv:11]: orcid correct: https://orcid.org/0000-0001-6622-7169

Comment (neteler) [reply contributors.csv:11]: What is wrong with the one I used? Maybe I don't see it...

**Comment by neteler** [comment]
Do we need a change in `gui/wxpython/gui_core/ghelp.py` to take care of the new column?

File: contributors.csv (hunk 1 above)
Comment (neteler) [reply contributors.csv:11]: ah, it *is* correct... ok.
(I suggest to now confirm all of them but rather only add missing ones)

**Comment by lucadelu** [comment]
Thanks Markus, mine is fine

**Comment by tmszi** [comment]
> Do we need a change in `gui/wxpython/gui_core/ghelp.py` to take care of the new column?

Yes we do, [patches.zip](https://github.com/OSGeo/grass/files/8027728/patches.zip). The contributors.csv file required changes `sed -e 's/,-$/,-,-/g' -e 's/yes,$/yes,-/g' -i contributors.csv`.

**Comment by neteler** [comment]
> Yes we do, [patches.zip](https://github.com/OSGeo/grass/files/8027728/patches.zip). The contributors.csv file required changes `sed -e 's/,-$/,-,-/g' -e 's/yes,$/yes,-/g' -i contributors.csv`.

Thanks so much, @tmszi - added to this PR.

**Review by tmszi — APPROVED** [review]
Looks good to me as expected (compiled, and tested).

wxgui_about_gg_contributors

**Pull request merged**